content/docs/command-reference/push.md

```diff
@@ -332,3 +332,32 @@ $ aws s3api list-object-versions --bucket mybucket
         }
     ]
 ```
+
+With `version_aware` enabled, `dvc push` will also modify <abbr>dvc files</abbr>
```

Comment:
I wonder if this should be part of the Description of the command (something like "Using with cloud versioned remotes .."